src/NewsService.Mappers/Models/NewsMapper.cs

```diff
@@ -19,7 +20,7 @@ public DbNews Map(News request)
                 Id = Guid.NewGuid(),
                 Content = request.Content,
                 Subject = request.Subject,
-                Pseudonym = request.Pseudonym,
+                Pseudonym = request.Pseudonym.Trim().Any() ? request.Pseudonym.Trim() : null,
```

Comment:
if pseudonym is null, you will get null ref exc

Reply:
fixed